lib/dal/src/audit_logging.rs

```diff
@@ -131,13 +131,37 @@ pub(crate) async fn publish_pending(
                 error!(?err, "audit logs shuttle error");
             }
 
-            match WsEvent::audit_logs_published(&ctx_clone_for_ws_event).await {
-                Ok(event) => {
-                    if let Err(err) = event.publish_immediately(&ctx_clone_for_ws_event).await {
-                        error!(?err, "error when publishing ws event for audit logs");
+            match ChangeSet::find(
+                &ctx_clone_for_ws_event,
+                ctx_clone_for_ws_event.change_set_id(),
+            )
+            .await
+            {
+                Ok(Some(change_set)) => {
+                    match WsEvent::audit_logs_published(
+                        &ctx_clone_for_ws_event,
+                        change_set.id,
+                        change_set.status,
+                    )
+                    .await
+                    {
+                        Ok(event) => {
+                            if let Err(err) =
+                                event.publish_immediately(&ctx_clone_for_ws_event).await
+                            {
+                                error!(?err, "error when publishing ws event for audit logs");
+                            }
+                        }
+                        Err(err) => error!(?err, "error when creating ws event for audit logs"),
                     }
                 }
-                Err(err) => error!(?err, "error when creating ws event for audit logs"),
+                Ok(None) => {
+                    trace!("skipping ws event creation for audit logs: no change set found")
+                }
```

Comment:
This is the case we are hitting. I am fairly confident we are running into the ephemeral change set here, but will need to follow up once we switch to PG-backed audit logs for the UI.